### PR TITLE
[MERGE] update the translation template and translation files for the module.

### PR DESCRIPTION
**Pull request by zaoral**
Task [vx#3357](https://www.vauxoo.com/#id=3357&view_type=form&model=project.task) HU [319](https://www.vauxoo.com/?debug=#id=319&view_type=form&model=user.story&action=726) Sprint 10

### CODE REVIEW

**Comment by zaoral** [comment]
The testing tools raise errors not related with the current change.

**Comment by zaoral** [comment]
@nhomar please do this merge.

File: purchase_requisition_for_everybody/i18n/es.po [hunk 1]
```diff
@@ -1,80 +1,83 @@
...
-msgstr "Confirmar pedido de compra"
+#: view:purchase.requisition:purchase_requisition_for_everybody.inherit_purchase_requisition_view1
+msgid "Requests for Quotation"
+msgstr "Solicitudes de presupuesto"
 
 #. module: purchase_requisition_for_everybody
 #: model:res.groups,name:purchase_requisition_for_everybody.purchase_requisition_manager_ev_groups
 msgid "Requisition - Manager"
-msgstr "Solicitud - Administrador"
+msgstr "Cotizaciones de Compra - Administrador"
```
Comment (nhomar) [inline purchase_requisition_for_everybody/i18n/es.po]: Requsition no es "Cotización de Compra" el término correcto es "Requisición"

File: purchase_requisition_for_everybody/i18n/es.po [hunk 2]
```diff
@@ -1,80 +1,83 @@
...
 
 #. module: purchase_requisition_for_everybody
-#: view:purchase.requisition:0
-msgid ""
-"purchase_requisition_for_everybody.purchase_requisition_manager_ev_groups"
-msgstr ""
-"purchase_requisition_for_everybody.purchase_requisition_manager_ev_groups"
+#: model:res.groups,name:purchase_requisition_for_everybody.purchase_requisition_user_ev_groups
+msgid "Requisition - User"
+msgstr "Cotizaciones de Compra - Usuario"
```
Comment (nhomar) [inline purchase_requisition_for_everybody/i18n/es.po]: Same than above.

File: purchase_requisition_for_everybody/i18n/es_MX.po [hunk 3]
```diff
@@ -1,16 +1,83 @@
...
+
+#. module: purchase_requisition_for_everybody
+#: view:purchase.requisition:purchase_requisition_for_everybody.inherit_purchase_requisition_view1
+msgid "Requests for Quotation"
+msgstr "Solicitudes de presupuesto"
+
+#. module: purchase_requisition_for_everybody
+#: model:res.groups,name:purchase_requisition_for_everybody.purchase_requisition_manager_ev_groups
+msgid "Requisition - Manager"
+msgstr "Cotizaciones de Compra - Administrador"
```
Comment (nhomar) [inline purchase_requisition_for_everybody/i18n/es_MX.po]: Igual

File: purchase_requisition_for_everybody/i18n/purchase_requisition_for_everybody.pot [hunk 4]
```diff
@@ -16,44 +16,43 @@ msgstr ""
 "Plural-Forms: \n"
 
 #. module: purchase_requisition_for_everybody
-#: view:purchase.requisition:0
-msgid "Reference"
+#: view:purchase.requisition:purchase_requisition_for_everybody.inherit_purchase_requisition_view1
+msgid "Approved by Supplier"
 msgstr ""
```
Comment (nhomar) [inline purchase_requisition_for_everybody/i18n/purchase_requisition_for_everybody.pot:22]: No traducido.

Comment (zaoral) [inline purchase_requisition_for_everybody/i18n/purchase_requisition_for_everybody.pot:22]: This is a template file and have not to be translated.

**Comment by zaoral** [comment]
@nhomar the changes you suggest were made. Please review this PR

**Pull request merged**